File: 프로그래머스/0/120885. 이진수 더하기/이진수 더하기.py
def solution(bin1, bin2):
    # 1. 이진수 문자열을 정수로 변환
    decimal_sum = int(bin1, 2) + int(bin2, 2)
    
    # 2. 정수를 이진수 문자열로 변환
    return bin(decimal_sum)[2:]

File: 프로그래머스/0/120885. 이진수 더하기/이진수 더하기.java
class Solution {
    public String solution(String bin1, String bin2) {
        String answer = "";
        
        int result = Integer.parseInt(bin1,2) + Integer.parseInt(bin2, 2);
        
        answer = Integer.toString(result, 2);
        
        System.out.println(result);
        System.out.println(Integer.toString(result, 2));
        
        return answer;
    }
}